metabase/metabase
# feat(sdk): generate sample react component with the embedding cli

Closes https://github.com/metabase/metabase/issues/46417

### Description

Generates example React components that demonstrates how to setup the embedding SDK.

### How to verify

- Run `yarn build-embedding-sdk-cli:watch` in one terminal, in your Metabase directory
- Create an empty React project, using `yarn create vite`
- Run `yarn add file:../metabase/resources/embedding-sdk` to add the dev version of the SDK
- Run `npx @metabase/embedding-sdk-react start`
- Go through the CLI's steps until the end. Watch the below demo video. Make sure to add a license key, either at the beginning or after the setup step.
- You should get the same result as in the demo video below.

### Demo

<div>
    <a href="https://www.loom.com/share/40edc8d096fe49e5ba1e6ea83631d9ff">
      <p>Embedding CLI - From Zero to Dashboard - Watch Video</p>
    </a>
    <a href="https://www.loom.com/share/40edc8d096fe49e5ba1e6ea83631d9ff">
      <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/40edc8d096fe49e5ba1e6ea83631d9ff-613f39c9b1ae5909-full-play.gif">
    </a>
  </div>
